recipe/meta.yaml

```diff
@@ -32,4 +32,4 @@ about:
 
 extra:
   recipe-maintainers:
-    -janschulz
\ No newline at end of file
+    - janschulz
```

Comment:
No, that was just the last drop... It was a pain to maintain, because of things like the conda-smithy create-github thing and so on. Now it's `@janschulz` on twitter and here :-)